frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segments_view.tsx

```diff
@@ -1195,6 +1581,22 @@ class SegmentsView extends React.Component<Props, State> {
     this.props.onUpdateSegmentGroups(newSegmentGroups, this.props.visibleSegmentationLayer.name);
   }
 
+  doesGroupHaveAnyMeshes = (groupId: number): boolean => {
+    const { visibleSegmentationLayer } = this.props;
+    if (visibleSegmentationLayer == null) return false;
+    const segmentGroup = this.getSegmentsOfGroup(groupId);
+    if (segmentGroup == null) return false;
+    const isosurfacesOfLayer =
+      Store.getState().localSegmentationData[visibleSegmentationLayer.name].isosurfaces;
+    let bool = false;
+    segmentGroup.forEach((segment) => {
```

Comment:
Sound like a good use for `Array.some(...)` 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some